trajectory.py

```diff
@@ -83,22 +88,6 @@ def take_step(self, step: np.ndarray):
         """
         self.positions.append(self.position + step)
 
-    def check_nucleus(
-        self, step, collision_style: str = "reflect"
-    ) -> Union[np.ndarray, bool]:
-
-        next_locus_extent = np.linalg.norm(self.position + step) + self.locus_radius
-
-        nuclear_check = self.nuclear_radius > next_locus_extent
-
-        if not nuclear_check and collision_style == "reflect":
-
-            overstep = next_locus_extent - self.nuclear_radius
-            return step - overstep * step
-
-        if not nuclear_check and collision_style == "reject":
-            return nuclear_check
-
```

Comment:
Consider restoring this and then letting it fork down an if statement tree depending on the user's choice of boundary handling

Reply:
As discussed, we will instead implement a check nucleus method that will perform the np.linalg.norm(pos+step)+locus_rad and simplify trajectory code

Reply:
new method trajectory.radius_post_step now does this calculation